specification/vmware/resource-manager/Microsoft.AVS/preview/2020-07-17-preview/vmware.json

```diff
@@ -0,0 +1,3923 @@
+        }
+      }
+    },
+    "ManagementCluster": {
+      "description": "The properties of a default cluster",
+      "required": [
+        "clusterSize"
+      ],
+      "properties": {
+        "clusterSize": {
+          "description": "The cluster size",
+          "type": "integer",
+          "format": "int32"
+        },
+        "clusterId": {
+          "description": "The identity",
+          "type": "integer",
+          "format": "int32",
+          "readOnly": true
+        },
+        "hosts": {
+          "description": "The hosts",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ClusterProperties": {
+      "description": "The properties of a cluster",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ManagementCluster"
+        }
+      ],
+      "properties": {
+        "provisioningState": {
+          "description": "The state of the cluster provisioning",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Cancelled",
+            "Deleting",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "ClusterProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "storagePolicy": {
+          "description": "The storage policy of the cluster",
+          "$ref": "#/definitions/StoragePolicy"
+        }
+      }
+    },
+    "PrivateCloudList": {
+      "description": "A paged list of private clouds",
+      "properties": {
+        "value": {
+          "description": "The items on the page",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/PrivateCloud"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ClusterList": {
+      "description": "A paged list of clusters",
+      "properties": {
+        "value": {
+          "description": "The items on a page",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Cluster"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "AdminCredentials": {
+      "description": "Administrative credentials for accessing vCenter and NSX-T",
+      "properties": {
+        "nsxtUsername": {
+          "description": "NSX-T Manager username",
+          "type": "string",
+          "readOnly": true
+        },
+        "nsxtPassword": {
+          "description": "NSX-T Manager password",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-secret": true
```

Comment:
`x-ms-secret` should probably be removed here to fix the check: `message: 'Secret property "nsxtPassword", cannot be sent in the response.'`.

Reply:
I made https://github.com/leedouglas/azure-rest-api-specs/pull/3 to remove this, @leedouglas.

Reply:
Please don't remove x-ms-secret.  Its valuable to have this knowledge in the spec.  If the linter is inappropriately failing due to this please either suppress or work with the SDK reviewer (@jianyexi ) to get the checks fixed.  Secrets are OK to return in a POST response.

Reply:
Okay, I'll revert that and see if I can get it suppressed.

Reply:
I reverted it locally, but I can not figure out how to suppress it when I run `oav validate-example $spec`. @jianyexi, do you know how to get that check fixed or suppressed?

Reply:
I tried to suppress it in https://github.com/Azure/azure-rest-api-specs/pull/11317/commits/d3a567f248d5198e467d24f91a44c1390caa52b3, but this did not work:
``` yaml
  - suppress: SECRET_PROPERTY
    from: vmware.json
    where:
      - $.definitions.AdminCredentials.properties.nsxtPassword
      - $.definitions.AdminCredentials.properties.vcenterPassword
    reason: Secrets are OK to return in a POST response.
```
@jianyexi, what should I do?

Reply:
@raych1  could you take a look ?

Reply:
@ctaggart , it has to be in `suppression` section as below.
```
## Suppression
``` yaml
directive:
  - suppress: SECRET_PROPERTY
    from: vmware.json
    where:
      - $.definitions.AdminCredentials.properties.nsxtPassword
      - $.definitions.AdminCredentials.properties.vcenterPassword
    reason: Secrets are OK to return in a POST response.
```